blockbase/inc/social-navigation.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+// We should only change the render of the navigtion block
+// to social links in the following conditions.
+function blockbase_condition_to_render_social_menu( $block ) {
```

Comment:
A theme may want to define a menu that follows these conditions to show up in the footer too for example, or a pre-footer template part. In some cases we may want to have the social navigation but maybe not always. The perfect example of this is actually Skatepark, with this PR in place, this is what I see when I insert the pre footer block pattern:

<img width="1310" alt="Screenshot 2021-10-04 at 10 30 22" src="https://user-images.githubusercontent.com/3593343/135818923-fb933887-6861-423f-a0d9-87a87cc04a38.png">